src/components/Presentation/index.jsx

```diff
@@ -51,9 +50,26 @@ const Presentation = ({
     return canvas
   }
 
+  const updateSlideWithNewResolution = (canvasObjects, slideWidth) => {
+    if (canvas.width != slideWidth) {
+      let scaleMultiplier = canvas.width / slideWidth
+      var objects = canvasObjects
```

Comment:
can we use es6 for these variables? we should avoid mixing es5 and es6

Reply:
Oops, lefts over from copy and paste